docs/docs/Configuration/configuration-authentication.md

```diff
@@ -4,149 +4,117 @@ sidebar_position: 0
 slug: /configuration-authentication
 ---
 
+The login functionality in Langflow serves to authenticate users and protect sensitive routes in the application.
 
+Starting from version 0.5, Langflow introduces an enhanced login mechanism that is governed by a few environment variables. This allows new secure features.
 
-:::info
+## Create a superuser and new users in Langflow
 
-This page may contain outdated information. It will be updated as soon as possible.
-
-:::
-
-
-
-
-## Sign Up and Sign In {#f480dac5d2094d75a433de0b8e195641}
-
-
----
+Learn how to create a new superuser, log in to Langflow, and add new users.
 
+1. Create a `.env` file and open it in your preferred editor.
 
-The login functionality in Langflow serves to authenticate users and protect sensitive routes in the application. Starting from version 0.5, Langflow introduces an enhanced login mechanism that is governed by a few environment variables. This allows new secure features.
+2. Add the following environment variables to your file.
 
+```bash
+export LANGFLOW_AUTO_LOGIN=False
+export LANGFLOW_SUPERUSER=admin
+export LANGFLOW_SUPERUSER_PASSWORD=securepassword
```

Comment:
The default user here was:
```
username: langflow
password: langflow
```

Adding the following variables didn't work to create the admin user, you need to manually create a username and password and then use the default account to give it admin permissions::
```
LANGFLOW_SUPERUSER=admin
LANGFLOW_SUPERUSER_PASSWORD=securepassword
```

Autologin is not working when adding the variable as false:
`LANGFLOW_AUTO_LOGIN=False`

Adding the variable for new users to be active also did not work:
`LANGFLOW_NEW_USER_IS_ACTIVE=False`

I will report this to see if it is a bug.

I have not been able to test this yet but I will check:
`LANGFLOW_SECRET_KEY=randomly_generated_secure_key`